### DISCUSSION TITLE
Alfred Gallery Integration

### DISCUSSION THREAD

**Discussion by makzsslm** [mr-pennyworth/alfred-better-dictionaries#30] [Ideas]
It would be useful to get automatic updates!

**Comment by mr-pennyworth** [comment]
Thanks @makzsslm! I'd love to do the gallery integration sometime, but I don't think presently the workflow is stable enough for it. There are many bugs and it is quite rough around the edges. In addition, this workflow uses https://github.com/mr-pennyworth/alfred-extra-pane to show the definitions in a sidebar, and the extra pane implementation has even more bugs!

But yup, certainly! If and when I think the pane and the workflow are polished / stable, it would be cool to submit it to the gallery.
